apps/portals/src/configurations/adknowledgeportal/routesConfig.ts

```diff
@@ -251,29 +239,22 @@ const routes: GenericRoute[] = [
     path: 'Explore',
     routes: [
       {
-        path: 'Programs',
-        routes: [
+        path: ':slug/',
```

Comment:
This new `/Explore/:slug/` route is necessary on all pages with an Explore section. It contains the RouteControlWrapper component.

The slug lets us show the component on any `/Explore/...` page without showing it on `/Explore/.../DetailsPage`. This behavior cannot be captured by the `exact` prop alone.